scripts/env-data.sh

```diff
@@ -25,8 +25,15 @@ fi
 if [ -z "${ALLOW_LISTING}" ]; then
 	ALLOW_LISTING=True
 fi
-if [ -z "${LOGGING}" ]; then
-	LOGGING=false
+if [ -z "${DISABLE_LOGGING}" ]; then
```

Comment:
What happens https://github.com/kartoza/docker-mapproxy/blob/master/scripts/start.sh#L56 now that logging env is gone

Reply:
Good you point it out. I made a typo there, but now it made me wonder how to combine disable logging and regular logging, so i pushed a change where if only logging is specified, disable logging will be the inversion of it, for the envsubst later. There are 4 combinations of logging and disable-logging but one is directly related to uwsgi settings and one to mapproxy's via log.ini so if this was to be introduced there would need to be some clarification in docs.